blocks/bricks/conv.py

```diff
@@ -1,5 +1,8 @@
-from theano.tensor.nnet.conv import conv2d, get_conv_output_shape
-from theano.tensor.signal.downsample import max_pool_2d, DownsampleFactorMax
+from theano import tensor
+from theano.tensor.nnet import conv2d
```

Comment:
No, the old `conv2d` is located in `theano.tensor.nnet.conv`. This one aliases to `theano.tensor.nnet.abstract_conv.conv2d`.